js/components/RequestFault.tsx

```diff
@@ -11,13 +11,21 @@ class RequestFault extends React.Component<IRequestFaultProps, any> {
 
     render() {
         if (this.props.response.mapElementInvokeResponses) {
+            let rootFaultsContent = [];
             const rootFaults = this.props.response.mapElementInvokeResponses[0].rootFaults || [];
 
+            if (!Array.isArray(rootFaults)) {
+                for (const key of Object.keys(rootFaults)) {
```

Comment:
I think ramda.values might simplify this.

Reply:
Yep, done